recipes/pylha/recipe.yaml

```diff
@@ -0,0 +1,45 @@
+context:
+  name: "pylha"
+  version: "0.4"
+  python_min: "3.7"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/pylha-${{ version }}.tar.gz
+  sha256: 911f9ff81343a28675933076f96026d2b8a53dbc0aabb953440ca56339c908cf
+
+build:
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - pyyaml
+
+tests:
+  - python:
+      imports:
+        - pylha
+      pip_check: true
+      python_version: ${{ python_min }}.*
+
+about:
+  home: https://github.com/DavidMStraub/pylha
+  summary: A Python package to convert data files in SLHA and similar formats to Python objects, JSON, or YAML.
+  license: MIT
+  license_file: LICENSE
+  repository: https://github.com/DavidMStraub/pylha
+
+extra:
+  recipe-maintainers:
+    - DavidMStraub
```

Comment:
👋 @DavidMStraub would you like to be added as a maintainer here in the same way as you were for `ckmutil` in PR https://github.com/conda-forge/staged-recipes/pull/30189 ?

Reply:
@DavidMStraub thanks for the :+1:, though for the linter to pass it requires that you leave add a comment in the PR mentioning that you accept (sorry).

Reply:
@DavidMStraub I'm going to remove this temporarily to get this merged, but will add you back in after the feedstock is created (unless you comment before it gets merged and then I can add it in here).